README.md

```diff
@@ -82,6 +82,16 @@ After the cloudtty is intergated to your own UI, it would look like:
 
   ![screenshot_png](https://github.com/cloudtty/cloudtty/raw/main/docs/snapshot.png)
 
+### Use custom cloudshell image
+
+Most users need more than just the basic Kubectl tools to manage their clusters. Users can customize images based on cloudshell base images. set the 'JobTemplate' image parameter to run your own cloudshell image when install cloudtty.
+
+```shell
+helm install cloudtty-operator --version 0.3.0 daocloud/cloudtty --set jobTemplate.image.registry=</your/registry> --set jobTemplate.image.repository=</your/repository> --set jobTemplate.image.tag=</your/tag>
```

Comment:
Can you add the ** sample Dockerfile **  to add a tool into the job images as well ?

Reply:
👌